android/app/src/main/java/com/zulipmobile/debug/MainActivity.java

```diff
@@ -0,0 +1,11 @@
+package com.zulipmobile.debug;
+
+import com.zulipmobile.ZulipActivity;
+
+/**
+ * Entry point activity for debug build
+ */
+
+public class MainActivity extends ZulipActivity {
```

Comment:
Clever solution. :-)

It's kind of mystifying to look at -- so we should be sure to have a comment in this and the other nearly-empty file. Basically most of the commit message on the commit that adds this can move into a comment.